zScanner/ViewModels/Documents/DocumentsListViewModel.swift

```diff
@@ -60,7 +60,7 @@ class DocumentsListViewModel {
     //MARK: Helpers
     let disposeBag = DisposeBag()
     
-    private var lastDepartmentSelected: String?
+    var lastDepartmentSelected: DepartmentButton?
```

Comment:
ViewModel cannot know about view. Pass DepartmentDomainModel or keep the string.